stubs/pywin32/win32comext/axdebug/documents.pyi

```diff
@@ -14,7 +14,7 @@ class DebugDocumentProvider(gateways.DebugDocumentProvider):
     def GetDocument(self): ...
 
 # error: Cannot determine consistent method resolution order (MRO) for "DebugDocumentText"
-class DebugDocumentText(gateways.DebugDocumentInfo, gateways.DebugDocumentText, gateways.DebugDocument):  # type: ignore[misc]
+class DebugDocumentText(gateways.DebugDocumentInfo, gateways.DebugDocumentText, gateways.DebugDocument):  # type: ignore[misc]  # pyright: ignore
```

Comment:
Which pyright error are we ignoring here? Can we add the error code?

Reply:
Same as above, no code, and if iirc, this does fail at runtime
> Cannot create consistent method ordering `Pylance`

I could create a feature request with pyright to add a code for "will fail at runtime". But this should rather be fixed upstream (and pywin32 already has accepted a couple fixes I found through type-checking).

And for the other case above, it's kindof a niche type-stub use-case, and is totally fixable by splitting off into another private/internal protocol (which I didn't do to keep changes minimal)